vector/src/main/java/fr/gouv/tchap/core/utils/RoomUtils.kt

```diff
@@ -33,21 +28,16 @@ enum class RoomTchapType {
 }
 
 object RoomUtils {
-    /**
-     * FIXME: Change roomAccessRule with real implementation
-     */
-    fun getRoomType(roomSummary: RoomSummary): RoomTchapType {
-        val roomAccessRule = RoomAccessRules.RESTRICTED
-
+    fun getRoomType(roomSummary: RoomSummary): TchapRoomType {
         return when {
-            roomSummary.isDirect    -> RoomTchapType.DIRECT
-            roomSummary.isEncrypted -> when (roomAccessRule) {
-                RoomAccessRules.RESTRICTED   -> RoomTchapType.PRIVATE
-                RoomAccessRules.UNRESTRICTED -> RoomTchapType.EXTERNAL
-                else                         -> RoomTchapType.UNKNOWN
+            roomSummary.isDirect    -> TchapRoomType.DIRECT
+            roomSummary.isEncrypted -> when (roomSummary.accessRules) {
+                RoomAccessRules.RESTRICTED   -> TchapRoomType.PRIVATE
+                RoomAccessRules.UNRESTRICTED -> TchapRoomType.EXTERNAL
```

Comment:
We should add the case RoomAccessRules.DIRECT -> TchapRoomType.DIRECT to cover all known cases
(see [ios code](https://github.com/dinsic-pim/tchap-ios/pull/388/files#diff-c3aee9b7ab4a72ff2af4460c83acd36f785245bae746005141870ee905d3e876R132)

I would keep "roomSummary.isDirect -> TchapRoomType.DIRECT", my suggestion is just to avoid missing one direct room